scripts/core/CONTRIB.rst

```diff
@@ -39,17 +39,66 @@ accepted into the project.
     Runtime team via the `GitHub issue tracker
     <https://github.com/oneapi-src/unified-runtime/issues/new>`_.
 
+Build Environment
+=================
+
+To be able to generate the source from the YAML files, the build environment
+must be configured correctly and all dependencies must be installed. The
+instructions for a basic setup are available in the `README
+<https://github.com/oneapi-src/unified-runtime/blob/main/README.md#building>`_.
+
+The following additional dependencies are required to support the ``generate``
+target:
+
+*    Doxygen (>= 1.8)
+
+*    The Python script requirements listed in `thirdparty/requirements.txt`_
+
+Doxygen can be installed via your system's package manager, e.g. on Ubuntu
+``sudo apt install doxygen``, or by downloading it from the Doxygen website. It
+must be available on the current ``PATH`` when the script is run.
+
+The simplest way to install the requirements for the script is using a Python
```

Comment:
we also have scripts for creating a container: https://github.com/oneapi-src/unified-runtime/blob/main/.github/docker/ubuntu-22.04.Dockerfile

Might also be worth mentioning. Using docker might be simpler for some.

Reply:
Good point, I've added a link to the README for the container scripts